alyeska/compose/compose_sh.py

```diff
@@ -166,7 +167,21 @@ def convert_yaml_to_sh(
     return shell_script
 
 
-def main():
+def init_flags(prefab_flags: List = None) -> None:
+    """ Initializes the flags for this tool.
+
+    Args:
+        prefab_flags (List, optional): A list of flags to parse. Useful for testing.
+
+    Returns:
+        None -- the now-parsed flags can be accessed through the FLAGS variable.
+    """
+    global FLAGS
+
+    # Don't re-parse flags
+    if FLAGS:
+        return
```

Comment:
More explicit:
```suggestion
        raise NotImplementedError("Cannot parse flags more than once.")
```

What do you think about raising an error if flags have already been parsed? This exit may cause some non-obvious behavior as is; e.g. If init_flags is called a second time with different arguments, the user won't know the flags haven't been updated.

Reply:
Fair point. Done.